spring-cloud-deployer-cloudfoundry/src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/ApplicationLogAccessor.java

```diff
@@ -54,29 +56,32 @@ public ApplicationLogAccessor(LogCacheClient logCacheClient) {
      * @return String containing the log information or empty string if no entries are available.
      */
     public String getLog(String deploymentId, Duration apiTimeout) {
+        logger.info("getLog:{}:{}", deploymentId, apiTimeout);
```

Comment:
Let's consider moving this to debug as it is really only interesting when debugging an issue. IMO

Reply:
Also, the format of `<method-name>:<arg0>:<argN>` is not too helpful and requires you to match the code to the log to understand what they are. I am in favor of using a sentence like "Retreiving log for deploymentId:{} w/ apiTimeout:{}"